docs/tutorials/start-a-private-network/keygen.md

```diff
@@ -31,95 +41,66 @@ Secret phrase `keep matrix knee meat awake frown rubber position federal easily
+| Key | Value |
+| --- | --- |
+| Secret phrase | `clip organ olive upper oak void inject side suit toilet stick narrow` |
+| Secret seed | `0x4bd2b2c1dad3dbe3fa37dc6ad5a4e32ddd8ad84b938179ac905b0622880e86e7` |
+| **SR25519** |  |
+| Public key | `0x9effc1668ca381c242885516ec9fa2b19c67b6684c02a8a3237b6862e5c8cd7e` |
+| SS58 Address | `5FfBQ3kwXrbdyoqLPvcXRp7ikWydXawpNs2Ceu3WwFdhZ8W4` |
+| **ED25519** | |
+| Public key | `0xb48004c6e1625282313b07d1c9950935e86894a2e4f21fb1ffee9854d180c781` |
+| SS58 Address | `5G9NWJ5P9uk7am24yCKeLZJqXWW6hjuMyRJDmw4ofqxG8Js2` |
 
 ### Pair 2
 
-#### SR25519
-Secret phrase `paper next author index wedding frost voice mention fetch waste march tilt` is account:
-  Secret seed:      0x4846fedafeed0cf307da3e2b5dfa61415009b239119242006fc8c0972dde64b0
-  Public key (hex): 0x74cca68a32156615a5923c67024db70da5e7ed36e70c8cd5bcf3556df152bb6d
-  Account ID:       0x74cca68a32156615a5923c67024db70da5e7ed36e70c8cd5bcf3556df152bb6d
-  SS58 Address:     5EhrCtDaQRYjVbLi7BafbGpFqcMhjZJdu8eW8gy6VRXh6HDp
-
-#### ED25519
-Secret phrase `paper next author index wedding frost voice mention fetch waste march tilt` is account:
-  Secret seed:      0x4846fedafeed0cf307da3e2b5dfa61415009b239119242006fc8c0972dde64b0
-  Public key (hex): 0x0fe9065f6450c5501df3efa6b13958949cb4b81a2147d68c14ad25366be1ccb4
-  Account ID:       0x0fe9065f6450c5501df3efa6b13958949cb4b81a2147d68c14ad25366be1ccb4
-  SS58 Address:     5CRZoFgJs4zLzCCAGoCUUs2MRmuD5BKAh17pWtb62LMoCi9h
+| Key | Value |
+| --- | --- |
+| Secret phrase |`paper next author index wedding frost voice mention fetch waste march tilt`|
+| Secret seed | `0x4846fedafeed0cf307da3e2b5dfa61415009b239119242006fc8c0972dde64b0` |
+| **SR25519** |  |
+| Public key | `0x74cca68a32156615a5923c67024db70da5e7ed36e70c8cd5bcf3556df152bb6d` |
+| SS58 Address | `5EhrCtDaQRYjVbLi7BafbGpFqcMhjZJdu8eW8gy6VRXh6HDp` |
+| **ED25519** | |
+| Public key | `0x0fe9065f6450c5501df3efa6b13958949cb4b81a2147d68c14ad25366be1ccb4` |
+| SS58 Address | `5CRZoFgJs4zLzCCAGoCUUs2MRmuD5BKAh17pWtb62LMoCi9h` |
 
 ### Pair 3
 
-#### SR25519
-Secret phrase `ridge easy shift marine leave vendor clever mammal clown input race hollow` is account:
-  Secret seed:      0xfe97888532db5ecfbf87206184a232c7fc336d7aadb776419b24e3c0ff70c31a
-  Public key (hex): 0x103aebd7ab1e971be446b1a13eb40541ec8ec162deb71b6ae594b3ce02ee5137
-  Account ID:       0x103aebd7ab1e971be446b1a13eb40541ec8ec162deb71b6ae594b3ce02ee5137
-  SS58 Address:     5CRz8J64ksqQ1DHuCUhyc5x1C2tC7exbQpkFVqEmfaG9RKdc
-
-#### ED25519
-Secret phrase `ridge easy shift marine leave vendor clever mammal clown input race hollow` is account:
-  Secret seed:      0xfe97888532db5ecfbf87206184a232c7fc336d7aadb776419b24e3c0ff70c31a
-  Public key (hex): 0x4822d2a0b3c5665050cd3470717439ca23a3c3387b65efcf6a9fbcc93571ea4a
-  Account ID:       0x4822d2a0b3c5665050cd3470717439ca23a3c3387b65efcf6a9fbcc93571ea4a
-  SS58 Address:     5DhHecmfy3hB9TpWJvCRXi1EwU3A4AwYvmCYJemsUUAcxf4V
-
-### Pair 4
```

Comment:
Why remove the additional pairs? I'm okay either way, just curious.

Reply:
It is because I feel having three sample key pairs are good enough for learning / playing ;) But I don't have a strong stance on this. If you feel strongly about this. Ask one more time and I will add them back ;)